Server/JokesOnYou.Web.Api/Startup.cs

```diff
@@ -51,6 +52,32 @@ public void ConfigureServices(IServiceCollection services)
             {
                 //appcontext base directory is where the app entry point assembly is (bin folder)
                 config.IncludeXmlComments(Path.Combine(AppContext.BaseDirectory, "JokesOnYou.Web.Api.xml"));
+
+                //add authorization option to Swagger UI
+                config.AddSecurityDefinition("Bearer", new OpenApiSecurityScheme
```

Comment:
Lets move this configuration in to the service extension class with a new method called "ConfigureSwagger" or something like that.

Reply:
@chrisK00, moved the logic to ServiceExtensions.cs